app/models/shipit/deploy_spec/file_system.rb

```diff
@@ -70,11 +70,11 @@ def cacheable_config
             'variables' => deploy_variables.map(&:to_h),
             'max_commits' => maximum_commits_per_deploy,
             'interval' => pause_between_deploys,
-            'retries_on_timeout' => retries_on_deploy_timeout,
+            'retries' => retries_on_deploy,
```

Comment:
I liked the idea of being able to specify the retries for timeouts and failures and errors separately, but we can leave that idea for later and if we see that it's needed.